AdminWebsite/AdminWebsite/ClientApp/src/app/booking/add-participant/add-participant.component.spec.ts

```diff
@@ -799,20 +793,6 @@ describe('AddParticipantComponent', () => {
                 component.validateJudiciaryEmailAndRole();
                 expect(component.errorJudiciaryAccount).toBeFalsy();
             });
-
-            it('should have errorJudiciaryAccount set to true if search service returns null and role is Panel Member', () => {
-                searchServiceSpy.searchJudiciaryEntries.and.returnValue(of(null));
-                role.setValue('Panel Member'); // TODO fix magic string
-                component.validateJudiciaryEmailAndRole();
-                expect(component.errorJudiciaryAccount).toBeTruthy();
-            });
-
-            it('should have errorJudiciaryAccount set to true if search service returns null and role is Winger', () => {
```

Comment:
For whenever we need to readd this functionality these tests can probably stay gone as it looks like the previous tests do the same thing with test cases.